src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -232,6 +278,448 @@ public void testMergePatientHistory() throws Exception {
 		assertThatObsoletePatientWasDeletedAndMergedIsTheActiveOne(mergedPatient, obsoletePatient);
 	}
 
+	@Test
+	public void testMgrGetPatients() throws Exception {
+		_setupTestPatient(false);
+		ArrayList<Patient> patients = patientBrowserManager.getPatient();
+		testPatient.check(patients.get(patients.size() - 1));
+	}
+
+	@Test
+	public void testMgrGetPatientsPageable() throws Exception {
+		_setupTestPatient(false);
+		ArrayList<Patient> patients = patientBrowserManager.getPatient(0, 10);
+		testPatient.check(patients.get(patients.size() - 1));
+	}
```

Comment:
Maybe with a loop we can insert more records than pagesize and test the resultset size?

Reply:
Done

Reply:
Thanks!